google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerCloudMonitoringExporter.java

```diff
@@ -84,6 +88,9 @@ static SpannerCloudMonitoringExporter create(
     if (monitoringHost != null) {
       settingsBuilder.setEndpoint(monitoringHost);
     }
+    if (Strings.isNullOrEmpty(universeDomain)) {
```

Comment:
```suggestion
    if (!Strings.isNullOrEmpty(universeDomain)) {
```

(And add a test that would catch this)

Reply:
Done